### ISSUE TITLE
Configure blog feeds full text

### ISSUE THREAD

**Issue by maiki** [writethedocs/www#328]
I prefer full text feeds. I didn't find any issues about it, so presuming the summary feed was just left at default. I tracked down [`blog_feed_fulltext`](http://ablog.readthedocs.io/manual/ablog-configuration-options/#confval-blog_feed_fulltext), hoping someone can approve that and add it to the config. :smiley:

**Comment by plaindocs** [comment]
Hi @maiki 

Sounds ok to me. Would you like to make a PR for the relevant config?

**Comment by maiki** [comment]
Sorry, I didn't lead in with that because I thought I'd need to run Sphinx locally to test for changes; [the feed](https://deploy-preview-329--dentist-canary-64055.netlify.com/blog/archive/atom.xml) looks a lot better now.

I added the parameter to limit the amount of entries. I set it to 10, as that covers about 2.5 months of posts at a time, and now that the feed is full text, keeping it populated with all posts + feed polling would cause unnecessary bandwidth usage.
